src/sentry/utils/env.py

```diff
@@ -9,7 +9,11 @@
 
 
 def in_test_environment() -> bool:
-    return "pytest" in sys.argv[0] or "vscode" in sys.argv[0]
+    return (
+        "pytest" in sys.argv[0]
+        or "vscode" in sys.argv[0]
+        or os.environ.get("SENTRY_IN_TEST_ENVIRONMENT") in {"1", "true"}
```

Comment:
@vartec are you sure we don't have any other existing flags we can use?

Reply:
@scefali  Alternative options:
- `SENTRY_ENVIRONMENT` — I cannot really use it, because it expects to be set to `"development"` without differentiating between test and not test. 
- `PYTEST_CURRENT_TEST` — doesn't work, because this is checked during setup, before it's in a test
- `"pytest" in sys.modules` — works, but seems risky, all it shows is that there is `import pytest` somewhere

I didn't find other options